lib/l10n/arb/app_de.arb

```diff
@@ -1402,7 +1402,7 @@
tically through technology (see Automatic Information Collection and Tracking) and other non-personal information we collect to improve our Application and to deliver a better and more personalized experience by enabling us to:\nEstimate our audience size and usage patterns.\nStore information about the child's preferences, allowing us to customize the content according to individual interests.\nWe use geolocation information we collect to determine whether the user is in a location where it’s possible to use the Application for donating.\u2028\nOur Practices for Disclosing Children's Information\nWe may disclose aggregated information about many of our users, and information that does not identify any individual or device. In addition, we may disclose children's personal information:\nTo third parties we use to support the internal operations of our Application.\nIf we are required to do so by law or legal process, such as to comply with any court order or subpoena or to respond to any government or regulatory request.\nIf we believe disclosure is necessary or appropriate to protect the rights, property, or safety of Givt, our customers or others, including to:\nprotect the safety of a child;\nprotect the safety and security of the Application; or\nenable us to take precautions against liability.\nTo law enforcement agencies or for an investigation related to public safety.\u2028\nIf Givt is involved in a merger, divestiture, restructuring, reorganization, dissolution, or other sale or transfer of some or all of Givt's assets, whether as a going concern or as part of bankruptcy, liquidation, or similar proceeding or event, we may transfer the personal information we have collected or maintain to the buyer or other successor.\u2028\nSocial Features \nThe Application allows parents to view information about their child’s donation activities and any good deeds that the child records, and parents may provide certain responses to this information.\u2028\nAccessing and Correcting Your Child's Personal Information\nAt any time, you may review the child's personal information maintained by us, require us to correct or delete the personal information, and/or refuse to permit us from further collecting or using the child's information. \u2028\nYou can review, change, or delete your child's personal information by:\nLogging into your account and accessing the profile page relating to your child.\nSending us an email at support@givt.app. To protect your and your child’s privacy and security, we may require you to take certain steps or provide additional information to verify your identity before we provide any information or make corrections.\u2028\nOperators That Collect or Maintain Information from Children\nGivt Inc. is the operator that collects and maintains personal information from children through the Application. Givt can be contacted at support@givt.app, by mail at 3343 Peachtree Rd NE 145-1032, Atlanta, GA 30326, or by phone at +1 918-615-9611.",
```

Comment:
Anything we wanna do with that url?

www.givt.app/privacy-policy-givt4kids/

I tested it but if you change it to: www.givt.app/privacy-policy-givt/
it still works

Reply:
i dont wanna go to poeditor again 🤮